crates/services/txpool_v2/src/pool_worker.rs

```diff
@@ -197,14 +198,17 @@ pub(super) enum PoolExtractBlockTransactions {
     },
 }
 
-pub(super) enum PoolRemoveRequest {
+pub(super) enum PoolTxUpdateRequest {
+    PreconfirmedTransactions {
```

Comment:
I think instead of doing that, the transaction pool should subscribe for all pre-confirmation that comes from the `TxStatusUpdater`. 

In that case you also don't need `SkippedTransactions`, because you will receive `PreConfirmationSqueezedOut`.